draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -790,7 +792,7 @@ Further details of the algorithm are given in the following sections.
 
 ## Message identifiers
 
-### Primary ID (required)
```

Comment:
Couldn't we just use "Primary ID - required" or something or does that muck it up too? I think it looses a lot of clarity without that in the title